DB/src/main/java/io/deephaven/db/v2/parquet/ParquetReaderUtil.java

```diff
@@ -268,11 +283,22 @@ public static void readParquetSchema(final String filePath, final ColumnDefiniti
 
         for (ColumnDescriptor column : schema.getColumns()) {
             currentColumn.setValue(column);
-            final LogicalTypeAnnotation logicalTypeAnnotation = column.getPrimitiveType().getLogicalTypeAnnotation();
-            final String colName = column.getPath()[0];
+            final PrimitiveType primitiveType = column.getPrimitiveType();
+            final LogicalTypeAnnotation logicalTypeAnnotation = primitiveType.getLogicalTypeAnnotation();
+            final String parquetColumnName = column.getPath()[0];
+            final String colName;
+            final String mappedName = readInstructions.getColumnNameMapping(parquetColumnName);
```

Comment:
I'm convinced that either this code is doing the mappings backwards, or the code in `TableManagementTools` (which grabs the reverse mappings) is.

Reply:
In the timeless words of Boog, the bear in "Open Season" (movie): "Naw, naw, cornflake. You got it all twisted!"
See, when you call read you specify /from/ what you expect to get from the file, /to/ what you want to show up in DH.  The schema reading code is doing that right now (no support for user provided instructions yet), but the idea.
BUT.
BUT.
When we are in the LocationProvider code, the location provider code is going to be called from the upper level DH layer and is going to be asked "gimme gimme for DH column X".  So LocationProvider here needs the /reverse/ mapping see?
Like my linear algebra professor used to say in college about "every finite dimensional vector space has an orthonormal basis", it is all quite obvious when you /don't/ think about it.